app/javascript/dashboard/components/ModalHeader.vue

```diff
@@ -33,4 +33,7 @@ export default {
 .wrap-content {
   word-wrap: break-word;
 }
+.page-sub-title {
+  color: var(--s-600);
```

Comment:
```suggestion
  color: var(--color-heading);
```

Do we need this style here?